sandbox-templates/mdx-evaluate-create-react-app/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Should we make all this simpler? Especially for a reproduction starter?

Reply:
I'm cautious.
This HTML file is generated by CRA, it generally isn't viewed or edited.
We could _potentially_ cut things back to:

```html
<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<title>React App</title>
</head>

<body>
	<noscript>
		You need to enable JavaScript to run this app.
	</noscript>
	<div id="root"></div>
</body>

</html>
```

since we don't strictly need the PWA focused tags, but it feels a bit odd to me, as I still see this mostly as a generated file.

Reply:
I’d do:

```html
<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>MDX + CRA</title>
  </head>
  <body>
    <noscript>
      You need to enable JavaScript to run this app.
    </noscript>
    <div id="root"></div>
  </body>
</html>
```

There’s some stuff in the earlier example that really isn’t needed; `lang=en` is useful, but folks forget to change it when their app isn’t in english, which is bad; title can be updated.
No strong opinions. But I’d prefer to have a good base than this autogenerated one.

Reply:
updated